internal/cobraprompt/prompt.go

```diff
@@ -119,6 +120,23 @@ func (co CobraPrompt) Run(ctx context.Context, root *cobra.Command, cnfg *hazelc
 		},
 	}))
 	co.GoPromptOptions = append(co.GoPromptOptions, SuggestionColorOptions...)
+	history := goprompt.NewHistory()
+	f, err := os.OpenFile(cmdHistoryPath, os.O_RDWR|os.O_APPEND|os.O_CREATE, 0666)
```

Comment:
Why do we create it with 0666 permissions? We create with 0600 in CreateMissingDirsAndFileWithRWPerms. Also, do we open a file for every command or do we open it once and reuse it? I did not get it via looking at the code.

Reply:
Great catch thanks! I changed permissions to `0600`. 
And as you said, we open the file once.